README.md

```diff
@@ -650,7 +650,24 @@ underscore if a separation is needed, such as: popanitem or pop_an_item
         
 Variables which are set once and never change value should be all upper case,
 if a seperator is needed use an underscore.
-      
+
+### Ignored Values ###
+
+The variable name `junk` should be used in cases where a returned valued must be
+stored but the variable itself is unneeded.
+
+  No:
+
+```Python
+for _ in range(10):     
+  do_something()
```

Comment:
Would you say that `i` is permissible for an unused iterator? This example might imply to people that it's not.